libs/deal-ticket/src/components/deal-ticket/deal-ticket-fee-details.tsx

```diff
@@ -244,55 +247,72 @@ export const DealTicketMarginDetails = ({
 
   return (
     <>
-      <KeyValue
-        label={t('Margin required')}
-        value={formatRange(
-          marginRequiredBestCase,
-          marginRequiredWorstCase,
-          assetDecimals
-        )}
-        formattedValue={formatRange(
-          marginRequiredBestCase,
-          marginRequiredWorstCase,
-          assetDecimals,
-          quantum
-        )}
-        labelDescription={MARGIN_DIFF_TOOLTIP_TEXT(assetSymbol)}
-        symbol={assetSymbol}
-      />
-      <KeyValue
-        label={t('Total margin available')}
-        indent
-        value={formatValue(totalMarginAvailable, assetDecimals)}
-        formattedValue={formatValue(
-          totalMarginAvailable,
-          assetDecimals,
-          quantum
-        )}
-        symbol={assetSymbol}
-        labelDescription={TOTAL_MARGIN_AVAILABLE(
-          formatValue(generalAccountBalance, assetDecimals, quantum),
-          formatValue(marginAccountBalance, assetDecimals, quantum),
-          formatValue(currentMargins?.maintenanceLevel, assetDecimals, quantum),
-          assetSymbol
-        )}
-      />
-      {deductionFromCollateral}
-      <KeyValue
-        label={t('Current margin allocation')}
-        indent
-        onClick={
-          generalAccountBalance ? () => setBreakdownDialog(true) : undefined
-        }
-        value={formatValue(marginAccountBalance, assetDecimals)}
-        symbol={assetSymbol}
-        labelDescription={MARGIN_ACCOUNT_TOOLTIP_TEXT}
-        formattedValue={formatValue(
-          marginAccountBalance,
-          assetDecimals,
-          quantum
-        )}
-      />
+      <Accordion.Root type="single" collapsible>
+        <Accordion.Item value="margin">
+          <KeyValue
+            id="margin-required"
+            label={
+              <Accordion.Trigger className={TOOLTIP_TRIGGER_CLASS_NAME}>
```

Comment:
<img width="410" alt="Screenshot 2023-09-04 at 11 33 37" src="https://github.com/vegaprotocol/frontend-monorepo/assets/16125548/12eb965d-7b78-4498-a170-1b5e27604b47">


works well,
but don't we get an icon trigger or something (like in market info) so that it looks like an expandable panel? 


<img width="387" alt="image" src="https://github.com/vegaprotocol/frontend-monorepo/assets/16125548/574287da-83b5-4c9c-9437-180307a8eb1d">

Reply:
There is separate story to fix this issue 
https://github.com/vegaprotocol/frontend-monorepo/issues/4673